tests/lay-rendition-flow-pre-pag/EPUB/content_004.xhtml

```diff
@@ -0,0 +1,13 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>rendition:flow pre-paginated</title>
+  </head>
+  <body>
+    <h1>rendition:flow with pre-paginated Content Test</h1>
+
+    <p>This test passes if the previous two pages render as fixed layout pages in their own viewport and require pagination.</p>
```

Comment:
Dumb (?) question: why would the previous two pages require pagination? They are very short. I haven't run this test but my mental image doesn't work. (If the test actually passes in some RSes that you've tried, I think we're good, but I like to understand.)